src/main/java/org/example/requests/PostsRequests.java

```diff
@@ -0,0 +1,77 @@
+package org.example.requests;
+
+import org.example.models.PostGET;
+import org.example.models.PostPOST;
+import org.example.models.PostsCollection;
+import org.example.models.error.ErrorModel;
+import org.example.models.error.UserErrorModel;
+import org.example.wrappers.RestRequest;
+import org.example.wrappers.RestWrapper;
+import org.springframework.http.HttpMethod;
+
+public class PostsRequests extends ModelRequest<PostsRequests> {
+    public PostsRequests(RestWrapper restWrapper) {
+        super(restWrapper);
+    }
+
+    public PostsCollection getPosts() {
+
+        RestRequest request = RestRequest.simpleRequest(HttpMethod.GET, "/post");
+        return restWrapper.processModel(PostsCollection.class, request);
+    }
+
+    public PostGET getInfoByPostId(String postId) {
+        RestRequest request = RestRequest.simpleRequest(HttpMethod.GET, "/post/{id}", postId);
```

Comment:
You can create a constant for "/post/{id}" and use it as path also for update & delete methods.